app/models/collection.rb

```diff
@@ -8,11 +8,7 @@ class Collection < ApplicationRecord
   has_and_belongs_to_many :depositors, class_name: 'User', join_table: 'depositors'
   has_and_belongs_to_many :reviewers, class_name: 'User', join_table: 'reviewers'
 
-  validates :contact_email, presence: true, format: { with: Devise.email_regexp }
-  validates :description, presence: true
-  validates :managers, presence: true
-  validates :name, presence: true
-  validates :access, presence: true
+  validates :contact_email, format: { with: Devise.email_regexp }, unless: -> { contact_email.blank? }
```

Comment:
Is the unless necessary?

Reply:
This is needed if we want to allow a draft collection or work to be saved with a blank email, but also want to verify that the format of the email is valid if is provided.

Reply:
doesn't the format only run if it's not nil?

Reply:
Not sure about nil, but it is an empty string when submitted on the form (and left blank by the user), which seems to run the format check.

Reply:
Looks like you're right.  Can you use `allow_blank: true` here instead? That seems to be more clear.

Reply:
good idea, that also lets us get back to the strict type check (#437)